RobotFramework/src/org/frc4931/robot/component/Motor.java

```diff
@@ -58,4 +58,52 @@ else if (direction > 0)
         else
             return Direction.STOPPED;
     }
+
+    /**
+     * Create a new Motor instance that is composed of two other motors that will be controlled identically. This is useful
+     * when multiple motors are controlled in the same way, such as on a {@link DriveTrain}.
+     * 
+     * @param motor1 the first motor, and the motor from which the speed is read; may not be null
+     * @param motor2 the second motor; may not be null
+     * @return the composite motor; never null
+     */
+    static Motor compose(Motor motor1, Motor motor2) {
+        return new Motor() {
+            @Override
+            public double getSpeed() {
+                return motor1.getSpeed();
+            }
+
+            @Override
+            public void setSpeed(double speed) {
+                motor1.setSpeed(speed);
+                motor2.setSpeed(speed);
+            }
+        };
+    }
+
+    /**
+     * Create a new Motor instance that is composed of three other motors that will be controlled identically. This is useful when
+     * multiple motors are controlled in the same way, such as on a {@link DriveTrain}.
+     * 
+     * @param motor1 the first motor, and the motor from which the speed is read; may not be null
+     * @param motor2 the second motor; may not be null
+     * @param motor3 the third motor; may not be null
+     * @return the composite motor; never null
+     */
+    static Motor compose(Motor motor1, Motor motor2, Motor motor3) {
+        return new Motor() {
+            @Override
+            public double getSpeed() {
+                return motor1.getSpeed();
+            }
+
+            @Override
+            public void setSpeed(double speed) {
+                motor1.setSpeed(speed);
+                motor2.setSpeed(speed);
+                motor3.setSpeed(speed);
+            }
+        };
+    }
```

Comment:
We could, but 1) it's not really necessary that we handle an arbitrary number of motors; 2) these implementations are more efficient than one based upon an array or a vararg; and 3) using an array makes it possible for someone to change the values within the array, whereas these implementations are definitely immutable.